idunn/blocks/wikipedia.py

```diff
@@ -57,6 +85,8 @@ def get_title_in_language(self, title, source_lang, dest_lang):
             },
             timeout=self.timeout,
         )
+        if resp.status_code in range(400, 500):
```

Comment:
isn't this too inefficient?  won't this generate 100 integer at each check ?

Reply:
indeed, done